libs/langchain-openai/src/embeddings.ts

```diff
@@ -65,12 +62,13 @@ export interface OpenAIEmbeddingsParams extends EmbeddingsParams {
  */
 export class OpenAIEmbeddings
   extends Embeddings
-  implements OpenAIEmbeddingsParams, AzureOpenAIInput
+  implements Partial<OpenAIEmbeddingsParams>
 {
-  modelName = "text-embedding-ada-002";
-
   model = "text-embedding-ada-002";
 
+  /** @deprecated Use "model" instead */
+  modelName: string;
```

Comment:
does this need a default value to not throw an error - or ability to make undefined

Reply:
You just have to set it in the constructor (which I do)